### PR TITLE
Additions

### PR DESCRIPTION
**Pull request by jdent02**
Metal BRDF
Bounce ui redo
Unlimited ray intensity

### CODE REVIEW

File: properties/materials.py [hunk 1]
```diff
@@ -64,26 +64,29 @@ class AppleseedMatLayerProps(bpy.types.PropertyGroup):
                                               ('metal_brdf', "Metal BRDF", ""),
                                               ('orennayar_brdf',
                                                "Oren-Nayar BRDF", ""),
-                                              ('plastic_brdf', "Plastic BRDF", ""),
-                                              ('sheen_brdf', "Sheen BRDF", ""),
+                                              #('plastic_brdf', "Plastic BRDF", ""),
```
Comment (dictoon) [inline properties/materials.py]: Hm... Revert these changes?

File: properties/scene.py [hunk 2]
```diff
@@ -74,6 +74,10 @@ def register(cls):
                                                         ('export_only', 'Export Files Only', '')],
                                                  default='render')
 
+        cls.threads_auto = bpy.props.BoolProperty(name="Auto Threads",
+                                                description="Automatically determine the number of rendering threads",
```
Comment (dictoon) [inline properties/scene.py]: Align arguments vertically.

File: properties/scene.py [hunk 3]
```diff
@@ -202,57 +206,57 @@ def register(cls):
                                                     description="Explicitly connect path vertices to light sources to improve efficiency",
                                                     default=True)
 
-        cls.max_bounces_unlimited = bpy.props.BoolProperty(name="",
+        cls.max_bounces_unlimited = bpy.props.BoolProperty(name="Unlimited",
```
Comment (dictoon) [inline properties/scene.py]: Align (everywhere).

File: properties/materials.py (hunk 1 above)
Comment (jdent02) [reply properties/materials.py]: I did that intentionally to keep the materials I haven’t added yet from showing up

File: properties/scene.py (hunk 2 above)
Comment (jdent02) [reply properties/scene.py]: Sorry about that

File: properties/scene.py (hunk 3 above)
Comment (jdent02) [reply properties/scene.py]: See apology above :)

Comment (dictoon) [reply properties/scene.py]: No worries :)

File: properties/materials.py (hunk 1 above)
Comment (dictoon) [reply properties/materials.py]: Oh ok, no problem then.

**Comment by jdent02** [comment]
Amended the last commit.
